### PR TITLE
Move old funnel people to TS and fix look

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

This improves the old funnel people component (particularly empty state) – which is still shown for most users.

| Before | After
| --- | --- |
| Before | After |

### CODE REVIEW

File: frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts [hunk 1]
```diff
@@ -45,7 +45,7 @@ export interface EntityFilterProps {
 // - filters
 // - setFilters
 // - typeKey
-export const entityFilterLogic = kea<entityFilterLogicType<BareEntity, EntityFilterProps, LocalFilter>>({
+export const entityFilterLogic = kea<entityFilterLogicType<BareEntity, LocalFilter>>({
```
Comment (paolodamico) [inline frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts:48]: Context on this change? It does look like indeed `EntityFilterProps` is part of the deps

File: frontend/src/scenes/funnels/People.tsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import { Link } from 'lib/components/Link'
+import { percentage, Loading } from 'lib/utils'
+import { EntityTypes } from '~/types'
+import './FunnelPeople.scss'
+import { Card } from 'antd'
+
+export function People(): JSX.Element | null {
+    const { stepsWithCount, peopleSorted, peopleLoading, areFiltersValid } = useValues(funnelLogic({}))
+
+    return !stepsWithCount || !areFiltersValid ? null : (
```
Comment (paolodamico) [inline frontend/src/scenes/funnels/People.tsx:13]: Code readability nit, maybe this is more clear?

```typescript
if (!stepsWithCount || !areFiltersValid) {
    return null
}
```

**Review by paolodamico — APPROVED** [review]
Minor comments but lgtm!

File: frontend/src/scenes/funnels/People.tsx (hunk 2 above)
Comment (Twixes) [reply frontend/src/scenes/funnels/People.tsx:13]: Why not

File: frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts (hunk 1 above)
Comment (Twixes) [reply frontend/src/scenes/insights/ActionFilter/entityFilterLogic.ts]: Ah, I think I had an old version of `kea-typegen` that removed this – upgraded and its back

**Pull request merged**